### PR TITLE
Add strings_not_translatable.xml and externalize App name

### PR DESCRIPTION
**Pull request by jakedot**
Created file for non-translatable strings and externalized the literal "WhereYouGo" in ApplicationManifest into string app_name.

See https://github.com/cgeo/WhereYouGo/pull/40#discussion_r392466416

### CODE REVIEW

**Comment by jakedot** [comment]
I haven't had the time to check if the specific string "WhereYouGo" can be externalized in other files. Maybe we want to wait with merging this PR until this is done.

**Comment by jakedot** [comment]
Externalized also usages of the MainActivity.APP_NAME constant. No other usages remaining.

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java [hunk 1]
```diff
@@ -317,7 +317,7 @@ private boolean isAnyCartridgeAvailable() {
             UtilsGUI.showDialogInfo(
                     MainActivity.this,
                     getString(R.string.no_wherigo_cartridge_available, FileSystem.ROOT,
-                            MainApplication.APP_NAME));
+                            getString(R.string.app_name)));
```
Comment (jakedot) [inline src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java]: I'm not sure why the APP_NAME string was used here. The no_wherigo_cartridge_available string does not seem to contain a format template usage for %2.

**Comment by jakedot** [comment]
@Lineflyer @bekuno can you review / merge this PR so I can continue #40 please?

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java (hunk 1 above)
Comment (bekuno) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java]: You are right. Maybe historic.
Please remove the parameter (and check ist).

**Comment by bekuno** [comment]
> 
> 
> @Lineflyer @bekuno can you review / merge this PR so I can continue #40 please?

Reviewed.
Others are fine.

**Comment by jakedot** [comment]
can you tell the ci bot it's
ok to test
?

**Comment by bekuno** [comment]
ok to test

**Comment by jakedot** [comment]
looks ok to me on my Samsung.

btw that text in german is not quite correct grammar wise:

https://github.com/cgeo/WhereYouGo/blob/0f70e60b57afa73bbef7944484a5a9ea45fa5015/src/main/res/values-de/strings.xml#L68-L75

> Kein[e] Wherigo-Cartridge [...]
> [...] Wenn du sich nicht sicher sind, [...]

But I guess this would belong in another PR.

File: src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java (hunk 1 above)
Comment (jakedot) [reply src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java]: removed and ready for merge from my perspective

**Pull request merged**